File: inOutPut/C++/1924.cpp
//2021.07.21
#include<iostream>
using namespace std;

int main(){
    int x, y;
    cin >> x >> y;
    int year = 2007;

    for(int i = 1; i < x; i++){
        switch (i)
        {
        case 2:
            y += 28;
            break;
        case 1:
            y += 31;
            break;
        case 3:
            y += 31;
            break;
        case 5:
            y += 31;
            break;
        case 7:
            y += 31;
            break;
        case 8:
            y += 31;
            break;
        case 10:
            y += 31;
            break;
        case 12:
            y += 31;
            break;
        default:
            y += 30;
            break;
        }
    }
    switch (y%7)
        {
        case 0:
            cout << "SUN";
            break;
        case 1:
            cout << "MON";
            break;
        case 2:
            cout << "TUE";
            break;
        case 3:
            cout << "WED";
            break;
        case 4:
            cout << "THU";
            break;
        case 5:
            cout << "FRI";
            break;
        default:
            cout << "SAT";
        }
    return 0;
}

File: inOutPut/Python/1924.py
#2021.07.21
x, y = input().split()
x = int(x)
y = int(y)

ans = [ "SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"]
for i in range(1, x):
    #1, 3, 5, 7, 8, 10, 12 = > 31
    if(i == 1 or i == 3 or i == 5 or i == 7 or i == 8 or i == 10 or i == 12):
        y += 31
    elif(i == 2):
        y += 28
    else:
        y += 30
print(ans[y%7])
